docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -42,44 +42,29 @@ mkdir packages
 cd packages
 mkdir theme
 cd theme
-yarn init
+yarn init -y
+touch index.js
```

Comment:
This allows the "Run the Site" section to work before we export the header component in a later section.

Reply:
touch isn't a valid command on windows. Don't think it matters much though, just pointing this out.